ReleaseProcessAutomation.UnitTests/Jira/JiraCredentialManagerTests.cs

```diff
@@ -0,0 +1,213 @@
+using System;
+using System.Net;
+using AdysTech.CredentialManager;
+using Moq;
+using NUnit.Framework;
+using ReleaseProcessAutomation.Configuration.Data;
+using ReleaseProcessAutomation.Jira.Authentication;
+using ReleaseProcessAutomation.Jira.CredentialManagement;
+using ReleaseProcessAutomation.Jira.ServiceFacadeImplementations;
+using ReleaseProcessAutomation.ReadInput;
+using Spectre.Console.Testing;
+
+namespace ReleaseProcessAutomation.UnitTests.Jira;
+
+[TestFixture]
+public class JiraCredentialManagerTests
+{
+  [SetUp]
+  public void Setup ()
```

Comment:
We should reorder these statements. Here are the general rules we try to adhere to:

- Constants > Static stuff > Instance stuff.
  - stuff is always: fields (including properties that act as state [auto properties]) > constructor > properties > methods
- public before private e.g. private instance fields then public instance fields
- Special NUnit methods like the setup and teardown usually come before any of the tests

This makes it easier to quickly find what you are looking for.